politeiad/backend/gitbe/gitbe_test.go

```diff
@@ -72,6 +72,11 @@ func TestAnchorWithCommits(t *testing.T) {
 	}
 	g.test = true
 
+	// Sets mime types map for validation
+	var mts = []string{
```

Comment:
This slice literal is written weird.  I would write it as either the declaration and the elements on the same line or all elements on a new line.

`mts := []string{"image/png", "text/plain", "text/plain; charset=utf-8"}`

or

```
mts := []string{
	"image/png",
	"text/plain",
	"text/plain; charset=utf-8",
}
```